fabtests/benchmarks/benchmark_shared.c

```diff
@@ -39,6 +39,14 @@
 #include "shared.h"
 #include "benchmark_shared.h"
 
+/*
+ * when the -j option is set, user supplied inject_size must be honored,
+ * even if the provider may return a larger value. This flag is used to
+ * distinguish between the '-j 0' option and no '-j' option at all. For
+ * both cases hints->tx_attr->inject_size is 0.
+ */
+static int option_j_set;
```

Comment:
Can we rename this to something like 'inject_set', rather than 'option_j'?  That would still work with longopts and be clearer to the reader (versus needing to track down what option j actually is).

Reply:
done.